tests/assert_extension_enabled.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Is this file called from anywhere? I don't see where it is referenced.

Reply:
this was actually here from when I was running the original test suite as part of our tests. now we can remove it. Thanks